src/Sulu/Bundle/AdminBundle/Resources/js/components/Overlay/Overlay.js

```diff
@@ -126,9 +125,9 @@ class Overlay extends React.Component<Props> {
 
         return (
             <Fragment>
-                <Backdrop onClick={onClose} open={visible} />
                 {visible &&
                     <Portal>
+                        <Backdrop />
```

Comment:
what is the intention behind removing the `onClick={onClose}` prop? 🤔

Reply:
It didn't work anyway, because the `Overlay` component was putting another div on top of it. Also, the `Dialog` component is also not closable by clicking on the `Backdrop`.

Reply:
alright - goot to know 🙂